core/http/clientWrapper.go

```diff
@@ -10,6 +10,17 @@ import (
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
 )
 
+const (
+	httpUserAgentKey = "User-Agent"
+	httpUserAgent    = "Elrond go SDK / 1.0.0 <Posting to nodes>"
```

Comment:
should we think on changing the version here?

Reply:
is just the agent version. Can be left as it is (1.0.0)
Getting the release version from the source repo is a rock in a shoe type of task. (binding @ compile time and other artifacts like this)